addons/spectator/functions/fnc_ui_draw3D.sqf

```diff
@@ -138,9 +132,7 @@ if !(GVAR(uiMapVisible)) then {
                 // Store grenade for next frame
                 _grenadesNew pushBack _x;
             };
-
-            nil // Speed loop
-        } count GVAR(grenadesToDraw);
+        } forEach GVAR(grenadesToDraw);
```

Comment:
If you swap the pushBack and drawIcon3D you will have a nil on the stack and can then use `count`